.github/workflows/test.yaml

```diff
@@ -7,3 +7,15 @@ jobs:
   unit-tests:
     uses: canonical/operator-workflows/.github/workflows/test.yaml@main
     secrets: inherit
+  look-for-outdated-code:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt update && sudo apt install python3-pip
+          pip3 install pyupgrade refurb
```

Comment:
TIL about pyupgrade and refurb - nice!

Reply:
I'm planning to discuss my experience with them at a demo soon